gourmet/reccard.py

```diff
@@ -1422,15 +1422,15 @@ def update_from_database (self):
             debug("Widget for %s"%c,5)
             model = self.rg.get_attribute_model(c)
             self.rw[c].set_model(model)
-            self.rw[c].set_text_column(0)
-            cb.setup_completion(self.rw[c])
+            # self.rw[c].set_text_column(0)
+            # cb.setup_completion(self.rw[c])
```

Comment:
Have we (temporarily) removed completion here? Can you add a comment why and what's the problem.

Reply:
That should not be there, good catch

Reply:
Fixed: the change is two-fold: first the API call has changed for setting the text column, but also the API call for getting the children when setting the callbacks.